syft/workers/base.py

```diff
@@ -1160,6 +1160,18 @@ def _get_msg(self, index):
 
         return sy.serde.deserialize(self.msg_history[index], worker=self)
 
+    def add_crypten_support(self):
+        """Add CrypTen specific methods"""
+        from syft.workers.worker_support import add_support
```

Comment:
Probably here the import could be global

Reply:
I left it into the method, because making it global it would create an import error.

```worker_support.py``` needs BaseWorker which is not defined :\